frontend/test/metabase/scenarios/question/custom_column.cy.spec.js

```diff
@@ -490,4 +490,58 @@ describe("scenarios > question > custom columns", () => {
     cy.findByText("Unknown Field: .5").should("not.exist");
     cy.findByRole("button", { name: "Done" }).should("not.be.disabled");
   });
+
+  describe("contentedtable field (metabase#15734)", () => {
+    beforeEach(() => {
+      // This is the default screen size but we need it explicitly set for this test because of the resize later on
+      cy.viewport(1280, 800);
+
+      openOrdersTable({ mode: "notebook" });
+      cy.findByText("Custom column").click();
+      popover().within(() => {
+        cy.get("[contenteditable='true']")
+          .as("formula")
+          .type("1+1")
+          .blur();
+      });
+      // Ugly hack without which the contenteditable field loses value making all subsequent repros impossible
+      cy.get("@formula")
+        .clear()
+        .type("1 + 1");
+      cy.findByPlaceholderText("Something nice and descriptive").type("Math");
+      cy.findByRole("button", { name: "Done" }).should("not.be.disabled");
+    });
+
+    it("should not accidentally delete CC formula value and/or CC name (metabase#15734-1)", () => {
+      cy.get("@formula")
+        .click()
+        .type("{movetoend}{leftarrow}{movetostart}{rightarrow}{rightarrow}")
+        .blur();
+      cy.findByDisplayValue("Math");
+      cy.findByRole("button", { name: "Done" }).should("not.be.disabled");
+    });
+
+    /**
+     * 1. Explanation for `cy.get("@formula").click();`
+     *  - Without it, test runner is too fast and the test resutls in false positive.
+     *  - This gives it enough time to update the DOM. The same result can be achieved with `cy.wait(1)`
+     */
+    it.skip("should not erase CC formula and CC name when expression is incomplete (metabase#15734-2)", () => {
+      cy.get("@formula")
+        .click()
+        .type("{movetoend}{backspace}")
+        .blur();
+      cy.findByText("Expected expression");
+      cy.findByRole("button", { name: "Done" }).should("be.disabled");
+      cy.get("@formula").click(); /* [1] */
+      cy.findByDisplayValue("Math");
+    });
+
+    it.skip("should not erase CC formula and CC name on window resize", () => {
```

Comment:
Forgot to add "(metabase#15734-3)" to the title, but don't want to change it now and wait 1 more hour for all tests to complete again.